documentation/docs/user_guide/constants/index.md

```diff
@@ -0,0 +1,16 @@
+# CABLE constants
+
+Table 1 lists the various types of constants used in CABLE. A list of the numerical values of each type can be accessed through the left navigation bar.
+
+
+## Table 1: Various constants used in CABLE 
+
+|   CABLE constants   	 |
+|------------------------|
+| Physical constants   	 |
+| Mathematical constants |
+| 			 |
+| 			 |
+| 			 |
+| 			 |
+|          	         |

```

Comment:
Any reason to have these extra empty lines here?

Reply:
Yes, as hinted to in the description, they are intended to accommodate in the future the names of the additional constants in CABLE/src/params but feel free to delete if they look ugly, I can reintroduce later on.

Reply:
I would prefer to add rows when we need them. 

```suggestion

```